extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -525,13 +524,11 @@ const AssetDiffs = ({ assetDiffs, icons }: AssetDiffsProps) => {
       default: {
         const code = "code" in diff.asset ? diff.asset.code! : "";
         const issuer = "issuer" in diff.asset ? diff.asset.issuer! : "";
-        const canonical = getCanonicalFromAsset(code, issuer);
-        const icon = icons[canonical];
         return (
           <div className="SignTransaction__AssetDiffRow">
             <div className="SignTransaction__AssetDiffRow__Asset">
               <AssetIcon
-                assetIcons={code !== "XLM" ? { [canonical]: icon } : {}}
+                assetIcons={code !== "XLM" ? icons : {}}
```

Comment:
a small adjustment here:

previously, until icons were loaded, the `assetIcons` param was `{"G123": undefined}`, which would cause `<AssetIcon` to show the "not found" image placeholder. If we pass it an empty object, (which `icons` will be until fetched), it will properly show the loading state